### ISSUE TITLE
Add Date of Issuance as Mandatory Field for Client Identifier	

### ISSUE THREAD

**Issue by santoshmath** [openMF/community-app#1846]
Reported by Ed Cable at https://mifosforge.jira.com/browse/MIFOSX-2404

Original Description:
While entering identity documents one should capture the Date of Issuance - add support for this mandatory date field to the Client Identifier.

![add_clientidentifier](https://cloud.githubusercontent.com/assets/23022667/22873192/5110a7ce-f1e4-11e6-8ea1-1c8db071b4c7.jpg)
![add_identifier1](https://cloud.githubusercontent.com/assets/23022667/22873190/51102484-f1e4-11e6-98af-b67f4f28e27d.png)
![addclient4](https://cloud.githubusercontent.com/assets/23022667/22873193/511097de-f1e4-11e6-919e-3d7d2c608451.png)
![addclientidentifier3](https://cloud.githubusercontent.com/assets/23022667/22873194/51112aaa-f1e4-11e6-9240-071889222b7d.png)
![view_clientidentifier1](https://cloud.githubusercontent.com/assets/23022667/22873191/51105242-f1e4-11e6-8be1-46e465854c30.jpg)

**Comment by santoshmath** [comment]
Commented by @Takahiro at https://mifosforge.jira.com/browse/MIFOSX-2404



Progress Report: I did rebase and passed the integrationTest on the backend. Currently, a related frontend PR (   MIFOSX-2405 CLOSED  ), which its backend merged first included in the rebase, is still under being reviewed so I am waiting for its merge to avoid merge conflict. I tested the frontend commits on my local environment and the implementation is working rightly with my part.

**Comment by santoshmath** [comment]
Commented by Dayna at https://mifosforge.jira.com/browse/MIFOSX-2404

Takahiro Ishikawa Is this still actively in development with you?

**Comment by santoshmath** [comment]
Commented by Takahiro at https://mifosforge.jira.com/browse/MIFOSX-2404

@dharp@mifos.org
Sorry to update the report. This issue is not actively in development with me.
Since I was away for a while from the active development and codebase monitoring, the progress I have made is not applicable for the current development. I changed the status to open.

**Comment by iamsk9** [comment]
@santoshmath  I would like to contribute to this issue. But my dobut is that I cannot find any **Date of Issuance** in the Add Client Identifier of the community app repo. So do I need to add **Date of Issuance** as a new field and then make it as mandatory.
I attached a screenshot 
![screenshot 15](https://cloud.githubusercontent.com/assets/10477029/23578095/dd19ae0e-00f4-11e7-885c-dea12fbbc5be.png)

**Comment by santoshmath** [comment]
Yes,  it was attempt made by Takahiro, and yet not completed. 'Date of Issuance' should be mandatory but may not be unique
